recipes/xerces-c/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: xerces-c
+  version: 3.1.3
+
+source:
+  url: http://mirrors.sonic.net/apache//xerces/c/3/sources/xerces-c-3.1.3.tar.gz # [not win]
+  fn: xerces-c-3.1.3.tar.gz # [not win]
+  md5: 70320ab0e3269e47d978a6ca0c0e1e2d # [not win]
+
+  url: http://mirrors.sonic.net/apache//xerces/c/3/sources/xerces-c-3.1.3.zip # [win]
+  fn: xerces-c-3.1.3.zip # [win]
+  md5: 4b3ab521a3854c774b267005fb4d8ebd # [win]
```

Comment:
It's almost never necessary to have win line endings.  Unless you've tested it and know it doesn't work, try for just the tarballs.  I can't recall any time it was necessary for me to use windows-specific source.